src/main/java/core/basesyntax/service/strategy/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.FruitStorage;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void operate(FruitTransaction fruitTransaction) {
+        String fruitType = fruitTransaction.getFruit();
+        if (FruitStorage.storageMap.get(fruitType) < fruitTransaction.getQuantity()) {
+            throw new InvalidDataException("Not enough fruit in the store");
```

Comment:
bad message 
I want to buy 10 bananas but the shop contains just 5
How will I know it? How will I know how many bananas I can buy?